fastify-autoload.d.ts

```diff
@@ -7,6 +7,7 @@ export interface AutoloadPluginOptions {
   dir: string
   ignorePattern?: RegExp | string
   includeTypeScript?: boolean
+  esmModules?: boolean
```

Comment:
This is redundant. “ESM” === “ECMA Script Modules”. So this code is saying “ECMA Script Modules Modules”.

Reply:
`esModules` sounds good for you?

Reply:
`loadESM: true`?